--- lyricue~	2010-04-01 00:38:48.000000000 +0300
+++ lyricue	2010-05-27 16:16:07.000000000 +0300
@@ -9889,7 +9889,11 @@
             my $tablename = $_;
             $tablename =~ s/^.*`(.*)`$/$1/g;
             $tablename =~ s/^"(.*)"$/$1/g;
-            my @fields = @{$db->selectall_arrayref("describe ".$tablename)};
+            my $tmpfields = $db->selectall_arrayref("describe " . $tablename);
+            if (!defined $tmpfields) {
+                next;
+            }
+            my @fields = @{$tmpfields};
             my $a= $fields[0]->[0];
             if ($a eq "verseid") {
                 my $query = "SELECT verse FROM ".$tablename." WHERE book=\"Bible\";";
