--- lyricue	2010-05-21 20:02:48.000000000 +0300
+++ lyricue	2010-05-21 21:22:19.045391671 +0300
@@ -1,4 +1,4 @@
-#!/usr/bin/env perl
+#!/usr/bin/perl
 my $ID = q$Id$;
 
 #****** lyricue/pod
--- lyricue_remote	2010-05-21 20:02:48.000000000 +0300
+++ lyricue_remote	2010-05-21 21:22:23.666223831 +0300
@@ -1,4 +1,4 @@
-#!/usr/bin/env perl
+#!/usr/bin/perl
 #****** lyricue_remote/setup
 # NAME
 #   Setup section
--- lyricue_server	2010-05-21 20:27:46.000000000 +0300
+++ lyricue_server	2010-05-21 21:22:29.429556938 +0300
@@ -1,4 +1,4 @@
-#!/usr/bin/env perl
+#!/usr/bin/perl
 my $ID = q$Id$;
 
 #****** lyricue_server/pod
